### PR TITLE
doc: add details of yearly awards program

### PR DESCRIPTION
**Pull request by mhdawson**
Add details based on recent CPC working session

Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
@jorydotcom @rginn PR that you can add the suggestion for how submissions will be reviewed/decided to.

File: programs/thank-you.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # Thank You Program
+
+The thank you program currently supports thank you's to contributors inthe
```
Comment (joesepi) [inline programs/thank-you.md:3]:
```suggestion
The thank you program currently supports thank yous to contributors in the
```

File: programs/yearly-awards.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Yearly collabor awards at OpenJS World
```
Comment (joesepi) [inline programs/yearly-awards.md:1]:
```suggestion
# Yearly collaborator awards at OpenJS World
```

**Review by joesepi — APPROVED** [review]
LGTM. Some minor fixes

**Review by dylans — APPROVED** [review]

**Review by ljharb — APPROVED** [review]

**Comment by mhdawson** [comment]
Agreed in the CPC meeting today that this can land.

@joesepi will follow up with @jorydotcom on a follow up PR to update to reflect any mismatch with what went out.

**Pull request merged**